worldengine/world.py

```diff
@@ -351,7 +350,7 @@ def sea_level(self):
     # Tiles around
     #
 
-    def on_tiles_around_factor(self, factor, pos, radius=1, action=None):
+    def on_tiles_around_factor(self, factor, pos, action, radius=1):
```

Comment:
```action=None``` seemed unsafe since that case wasn't even handled by the function. In other words: ```action``` was a required parameter disguised as an optional one.